### PR TITLE
[asmjit] update to current version

### PR DESCRIPTION
**Pull request by jsmolka**
Updates asmjit to current version.

### CODE REVIEW

**Comment by jsmolka** [comment]
Sadly I am not quite sure what the errors are about.

File: ports/asmjit/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (LilyWangL) [inline ports/asmjit/portfile.cmake]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

File: ports/asmjit/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
...
   OUT_SOURCE_PATH SOURCE_PATH
   REPO asmjit/asmjit
-  REF 761130b1d8f32b5d3d612d285664fcfef5258149
-  SHA512 a86fd58ba0c8bc81ec575e86a9acdf4a11f2acc9c2facd2a0a8512cffa9ee6fc0bd877a1f33fb52f8f510eff1de654b45cd4f5f5a18c5252ecae22a92db6e93e
+  REF 7e164e3edeefb76d8a2da4fbe84957ece0d07a13
+  SHA512 19d0a7f7a7cb4e6bd6c03f2e29ab012edf67ba4ba92789fd81e71a4937f1271a5124fa9c02d40d202bad7785b9e5ae21a3a72048cdeb8781c1b927c9717669c8
   HEAD_REF master
 )
```
Comment (LilyWangL) [inline ports/asmjit/portfile.cmake]: Please remove the extra space lines in _portfile.cmake_.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

**Comment by LilyWangL** [comment]
Hi @jsmolka , thanks for your updates!

Please fix the above changes. The failures on Windows are not related to this changes. It should not block to merge this PR.

**Comment by jsmolka** [comment]
@LilyWangL I've already deleted my fork of vcpkg and don't know if it's possible to fix the issues online.

**Comment by LilyWangL** [comment]
Hi @jsmolka, due to the branch of this PR has been removed, you need do follow these steps:
1. Create a new branch on your fork vcpkg.
2. Change the related file.
3. Commit your changes.
4. Close this PR and create a new PR to commit this changes.

**Comment by jsmolka** [comment]
I will do that later today, thank you.

**Pull request closed without merging**